Album/ViewControllers/ProfileControllers/FriendProfileViewController.m

```diff
@@ -53,27 +53,29 @@ - (void) fetchProfile {
     }
 }
 
+- (void) fetchFriendStatus {
+    
+}
+
```

Comment:
do we need to mark this as WIP? 
Is this used?

Reply:
This is used in the branch I'm working today (7/14)... I guess I added the method before switching to a new branch. Should I remove it?